src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -174,6 +175,7 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
       // Currently only supported sorting field is 'date_added'
     };
 
+    const userTeams = new Set(teams.filter(({isMember}) => isMember).map(({id}) => id));
```

Comment:
not sure if this should be some sort of utility function, or where to put it if it were a util function